__tests__/implementation.test.js

```diff
@@ -292,69 +292,6 @@ test('batch endpoint (multiple requests)', async () => {
     });
 });
 
-test('batch endpoint that throws errors', async () => {
```

Comment:
Removing this since it's the same test as https://github.com/Yelp/dataloader-codegen/blob/3c4bcb19757ad3c136582b9e4eaeb0fc1978d2b4/__tests__/implementation.test.js#L421